.circleci/config.yml

```diff
@@ -284,6 +301,7 @@ executors:
     docker:
       - image: << parameters.image >>
     working_directory: << parameters.working-dir >>
+    resource_class: large
```

Comment:
we were running at 100% of cpu on the linux build when generating artifacts so I bumped it up...we still hit 100% cpu, but for like 50% of the time we were previously and get more memory free.